Source/WTF/wtf/FastMalloc.h

```diff
@@ -63,17 +83,33 @@ namespace WTF {
 //
 // - WTF_MAKE_TZONE_ALLOCATED(ClassName)
 // - WTF_MAKE_TZONE_ALLOCATED_EXPORT(ClassName, exportMacro)
-//     class / struct is allocated from a fixed set of shared libpas heaps with the same size and alignment. Each set of shared heaps is know as a
-//     TZoneTypeBuckets. The particular bucket, is selected using SHA 256 hashing using a process startup value along with particulars of the class / struct
-//     being allocated. This sharing provides protections from type confusion and use-after-free bugs, but with less memory address space overhead of
+// - WTF_MAKE_STRUCT_TZONE_ALLOCATED(ClassName)
+// - WTF_MAKE_TZONE_ALLOCATED_TEMPLATE(ClassName)
+// - WTF_MAKE_TZONE_ALLOCATED_TEMPLATE_EXPORT(ClassName, exportMacro)
+// - WTF_MAKE_COMPACT_TZONE_ALLOCATED(ClassName)
+// - WTF_MAKE_COMPACT_TZONE_ALLOCATED_EXPORT(ClassName, exportMacro)
+//     class / struct / template is allocated from a fixed set of shared libpas heaps with the same size and alignment. Each set of shared heaps is
+//     known as a TZoneTypeBuckets. The particular bucket is selected using SHA 256 hashing on a process startup value along with particulars of the
```

Comment:
The SHA256 comment no longer true.

Reply:
It still applies.  The SHA256 is used to compute the `m_tzoneKeySeed`, which is used in the selection of each bucket.  It's just that the selection uses a WeakRandom using that `m_tzoneKeySeed` and size info, etc., instead of another SHA256.